src/main/webapp/WEB-INF/view/searchResults.jsp

```diff
@@ -55,24 +57,29 @@
 					</c:if>
 					<div class="description">${doc.content_description}</div>
 				</div>
-				<div class="site text-truncate">
-					<cite>${f:h(doc.site_path)}</cite>
-					<c:if test="${doc.has_cache=='true'}">
-						<small class="d-none d-lg-inline-block"> <la:link
-								href="/cache/?docId=${doc.doc_id}${appendHighlightParams}"
-								class="cache">
-								<la:message key="labels.search_result_cache" />
-							</la:link>
-						</small>
-					</c:if>
-					<c:if test="${doc.similar_docs_count!=null&&doc.similar_docs_count>1}">
-						<small class="d-none d-lg-inline-block"> <la:link
-								href="/search?q=${f:u(q)}&ex_q=${f:u(queryEntry.value)}&sdh=${f:u(fe:sdh(doc.similar_docs_hash))}${fe:facetQuery()}${fe:geoQuery()}">
-								<la:message key="labels.search_result_similar"
-											arg0="${fe:formatFileSize(doc.similar_docs_count-1)}" />
-							</la:link>
-						</small>
-					</c:if>
+				<div class="site">
+					<div class="title-container" style="overflow:unset;">
+						<cite>${f:h(doc.site_path)}</cite>
+						<i class="clipboard-copy fa fa-copy" data-clipboard-text="${doc.url_link}"></i>
```

Comment:
Please add only this line.
Other changes are difficult at this time.

Reply:
I apologize for my various inadequacies.
Please consider again.

Reply:
It's difficult to apply this change... 
- UI is changed on some devices
- style attribute needs to be removed

Reply:
I have Windows 10 21H2(64bit) Chrome 100.0.4896.127(64bit), Firefox 099.0.1(64bit),Internet Explore 11 and it appears to be working fine.

I looked at the Chrome Developer tool with the SmartPhone screen size and could not find any particular problem.

The only thing I can think of is that the clipboard copy is positioned on the right side of the image below.
![image](https://user-images.githubusercontent.com/15380342/166442137-a1b57d3a-f8fd-4905-ba31-d4ef752a9b9b.png)

Reply:
I found the clipboard copy icon in Internet Explore does not work...
I try to fix it. My apologies.

Reply:
<ul>
  <li>UI is changed on some devices</li>
I was using syntax in search.js that is not supported by some browsers, such as const.
I rewrote it as var, etc. and confirmed that it works correctly in Internet Explorer.
<li>style attribute needs to be removed</li>
I have removed the style addition.
</ul>

Thank you very much for your kind response.
Please review it again.

Thanks

Reply:
Thank you for fixing it.
Hmm, I'll consider UI with a clipboard in 14.2 release...

Reply:
I would appreciate it very much.
Thank you very much for your kind attention to this request.